apps/svelte.dev/src/hooks.server.js

```diff
@@ -55,8 +55,8 @@ export async function handle({ event, resolve }) {
 		redirect(307, destination);
 	}
 
-	// For REPL. For some reason, the repl/+page.server.ts file is not working, so
-	// we are doing the redirect here
+	// For REPL. In SvelteKit, redirects can also be handled in +page.server.js,
+	// for example, see routes/examples/[...path]/+page.server.ts file.
```

Comment:
that comment was correct before, it talks about the fact that for some mysterious reason putting the reidrect into `rep/+page.server.ts` doesn't work.
```suggestion
	// For REPL. For some reason, the repl/+page.server.ts file is not working, so
	// we are doing the redirect here
```